tests/test_serializer_json.py

```diff
@@ -16,14 +16,16 @@
 from invenio_records import Record
 from marshmallow import Schema, fields
 
+from invenio_records_rest.schemas.fields import \
+    PersistentIdentifier as PIDField
 from invenio_records_rest.serializers.json import JSONSerializer
 
 
 def test_serialize():
     """Test JSON serialize."""
     class TestSchema(Schema):
         title = fields.Str(attribute='metadata.mytitle')
-        id = fields.Str(attribute='pid.pid_value')
+        id = PIDField(attribute='pid.pid_value')
```

Comment:
Shouldn't it be like this even [here](https://github.com/inveniosoftware/invenio-records-rest/pull/224/files#diff-e4dfb0584ad0d70acf59d636a0c8ff24R42)?